docs/wiki/[User-Guide]-Quick-Start-Phase-2-Azure-DevOps.md

```diff
@@ -144,12 +146,15 @@ Although you can just run `Deploy-Accelerator` and fill out the prompted inputs,
     | `allow_storage_access_from_my_ip` | `TF_VAR` | `false` | This controls whether to allow access to the storage account from your IP address. This is only needed for trouble shooting. This only applies if you have `use_private_networking` set to `true`. This defaults to `false`. |
     | `apply_approvers` | `TF_VAR` | `<email-address>` | This is a list of service principal names (SPN) of people you wish to be in the group that approves apply of the Azure landing zone module. This is an array of strings like `["abc@xyz.com", "def@xyz.com", "ghi@xyz.com"]`. You may need to check what the SPN is prior to filling this out as it can vary based on identity provider. Use empty array `[]` to disable approvals. Note if supplying via the user interface, use a comma separated string like `abc@xyz.com,def@xyz.com,ghi@xyz.com`. |
     | `create_branch_policies` | `TF_VAR` | `true` | This controls whether to create branch policies for the repository. This defaults to `true`. |
+    | `architecture_definition_name` | `TF_VAR` | N/A | This is the name of the architecture definition to use when applying the ALZ archetypes via the architecture definition template. This is only relevant to starter modules used by Microsoft Cloud for Industry, such as the `sovereign_landing_zone` starter module. This defaults to `null`. |
```

Comment:
I know we pulled the "Microsoft Cloud for Sovereignty" language out in favor of "Sovereign Landing Zone". Is a customer coming here first going to have any context for "Microsoft Cloud for Industry" or should we drop this part of the description?